modules/core/src/opencl/runtime/opencl_core.cpp

```diff
@@ -143,21 +161,19 @@ static void* GetProcAddress(const char* name)
         cv::AutoLock lock(cv::getInitializationMutex());
         if (!initialized)
         {
-            const char* path = "libOpenCL.so";
             const char* envPath = getenv("OPENCV_OPENCL_RUNTIME");
             if (envPath)
-                path = envPath;
-            handle = dlopen(path, RTLD_LAZY | RTLD_GLOBAL);
-            if (handle == NULL)
             {
-                if (envPath)
+                handle = GetHandle(envPath);
+		if (!handle)
```

Comment:
Currently there is indentation with 2 tabs (try to select that via GitHub Web UI).